### PR TITLE
Merge Main into release

### PR DESCRIPTION
**Pull request by MichaelSwigerAtBentley**
Merges changes into release/4.x branch for 4.7 release

imodel-native-internal: https://github.com/iTwin/imodel-native-internal/pull/530

### CODE REVIEW

**Comment by GintV** [comment]
@MichaelSwigerAtBentley could this be hold off until #778 is merged and Core 4.6.x patch is released with it? We need this to be on 4.6.x to avoid any more regressions for our hotfix release.

cc @aruniverse @grigasp

**Comment by DanRod1999** [comment]
> @MichaelSwigerAtBentley could this be hold off until #778 is merged and Core 4.6.x patch is released with it? We need this to be on 4.6.x to avoid any more regressions for our hotfix release.
> 
> cc @aruniverse @grigasp

is updating to 4.7 not an option? Sorry I don't know the full context, so I might be missing something

**Comment by GintV** [comment]
> > @MichaelSwigerAtBentley could this be hold off until #778 is merged and Core 4.6.x patch is released with it? We need this to be on 4.6.x to avoid any more regressions for our hotfix release.
> > cc @aruniverse @grigasp
> 
> is updating to 4.7 not an option? Sorry I don't know the full context, so I might be missing something

Just want to avoid any more regressions. As I understand 4.6.2 was already released with #778 , thanks!

**Review by DanRod1999 — APPROVED** [review]

**Pull request merged**
